infor-design/enterprise
# 746 - Fix parent and page shifts with busyindicator

**Explain the _details_ for making this change. What existing problem does the pull request solve?**
Fixed the whole page and parent div was shifts when active with Busy Indicator.

**Related github/jira issue (required)**:
Closes infor-design/enterprise-ng#746

**Steps necessary to review your pull request (required)**:
- Pull this branch and build/run the demo app
- Navigate to: http://localhost:4000/components/busyindicator/test-on-maincontent.html
- Click on button `Start Busy Indicator`
- See this button should not change it's position and page should not shift

Test on angular side:
- Pull this branch and build/run the demo app
- Pull and build NG branch ([master](https://github.com/infor-design/enterprise-ng))
- Copy and replace "dist" folder from `enterprise` to `⁨enterprise-ng⁩/node_modules⁩/ids-enterprise`
- Change line# 21-23 as below, in `enterprise-ng/src/app/app.component.html`
```html
<div id="maincontent" class="page-container scrollable" role="main" soho-busyindicator [activated]="active$ | async" [blockUI]="true">
  <button soho-button="primary" (click)="click()">Start Busy Indicator</button>
  <router-outlet></router-outlet>
</div>
```
- Add below as line numbers, in `enterprise-ng/src/app/app.component.ts`
```js
// add at line# 12
import {BehaviorSubject, Subject} from 'rxjs';

// add at line# 40
public active$: BehaviorSubject<boolean> = new BehaviorSubject<boolean>(false);

// add at line# 80
click() {
  this.active$.next(!this.active$.getValue());
  setTimeout(() => this.active$.next(false), 5 * 1000);
}
```
- Navigate to http://localhost:4200/ids-enterprise-ng-demo/datagrid-dynamic or any other page in demo app
- Click on top button `Start Busy Indicator`
- See this button should not change it's position and page should not shift

**Included in this Pull Request**:
